src/components/NonProfitApply/NonProfitApply.js

```diff
@@ -35,6 +38,8 @@ import {
 
 
 import LoginOrRegister from "../LoginOrRegister/LoginOrRegister";
+import ReactPixel from 'react-facebook-pixel';
```

Comment:
If we're going to double-down on marketing, we want to see how far people are getting with filling out these forms